exchanges/bybit/bybit_wrapper.go

```diff
@@ -1485,7 +1485,7 @@ func (by *Bybit) GetServerTime(ctx context.Context, _ asset.Item) (time.Time, er
 	return info.TimeNano.Time(), err
 }
 
-// transformInstrumentInfoSymbol converts GetInstrumentInfo symbol to one stored in config with proper delimiters
+// transformSymbol converts GetInstrumentInfo symbol to one stored in config with proper delimiters
```

Comment:
🗻 Out-Of-Scope request: This probably shouldn't be returning a string, and instead return a `currency.Pair` with delimiter set, which the comsumer can call `String` on. Otherwise we're just being lossy in the decisions this function just made about what the base and quote is, what the delimiter should be, etc.
Moreover, that leads to the question: Isn't this an `UnmarshalJSON` in disguise ?

🐒 Nitpick: If we don't go full nuclear on this, as outlined above, we can still clarify the comment, since we're not really doing anything with comments.
```suggestion
// transformSymbol returns a symbol with a delimiter added if missing
// * Spot and Coin-M add "_"
// * Options, USDC-M USDT-M add "-"
// * CrossMargin is left without a delimiter
```